modules/datahandling/userrequests.go

```diff
@@ -234,9 +235,43 @@ func (f userProjectsRequest) process(db dbfs.DBFS) ([]dhClosure, error) {
 			Status: messages.StatusPartialfail,
```

Comment:
if we're now returning `[]projectLookupResult{}` (on line 240) then this should be a full fail as opposed to a partial fail, because now we return nothing to the client whereas before we forwarded as much as we could.
Is this the right logic? what happens if a user is never able to get any projects because of some db error affecting one of them? Maybe we still need to look up the data on the ones we can.